src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -35,6 +35,8 @@ private void CheckDisposedOrStarted()
             }
         }
 
+        public static bool IsSupported => true;
```

Comment:
I will add it, but could you please clarify what are the rules about them?
I see a lot of existing public properties without comments e.g. [Expected100ContinueTimeout](https://github.com/alnikola/runtime/blob/de8d6455de23e65db0c3e07724c82ab9021e02f4/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs#L264) and AFAIK the xml doc content **doesn't** get automatically copied to dotnet/dotnet-api-docs repo. Previously, I had to manually create PR in the doc repo for new members (see [HttpRequestException.StatusCode](https://github.com/dotnet/dotnet-api-docs/pull/4341))

Reply:
+ @carlossanlop

Reply:
(Note the comment the bot added to the PR: https://github.com/dotnet/runtime/pull/40271#issuecomment-668079260. The XML comments on new APIs are then ported to the docs.)

Reply:
This is a strange API. What value are we going to return for example on iOS where the API does not work fully and we'll discourage usage of it (probably using some analyzers) because it's inefficient?